datafusion/core/tests/physical_optimizer/enforce_distribution.rs

```diff
@@ -394,29 +394,32 @@ fn test_suite_default_config_options() -> ConfigOptions {
     config
 }
 
-/// How the optimizers are run.
 #[derive(PartialEq, Clone)]
-enum DoFirst {
-    /// Runs: (EnforceDistribution, EnforceDistribution, EnforceSorting)
+enum Run {
     Distribution,
-    /// Runs: (EnforceSorting, EnforceDistribution, EnforceDistribution)
     Sorting,
 }
 
+/// Standard sets of the series of optimizer runs:
+const DISTRIB_DISTRIB_SORT: [Run; 3] =
+    [Run::Distribution, Run::Distribution, Run::Sorting];
+const SORT_DISTRIB_DISTRIB: [Run; 3] =
+    [Run::Sorting, Run::Distribution, Run::Distribution];
```

Comment:
Equivalent to main: https://github.com/apache/datafusion/blob/9a4c9d598ca93ef66ba02067cc1497d53c28cddc/datafusion/core/tests/physical_optimizer/enforce_distribution.rs#L471-L481